forte/data/vocabulary.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # pylint: disable=line-too-long
+from __future__ import annotations
```

Comment:
Same as in `base_extractor.py`. The purpose is to fix this problem https://stackoverflow.com/questions/15853469/putting-current-class-as-return-type-annotation.